src/main/java/com/example/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,11 +1,18 @@
 package com.example.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
 public record CreateBookRequestDto(
+        @NotNull
         String title,
+        @NotNull
         String author,
+        @NotNull
```

Comment:
You can add pattern to validate isbn

Reply:
what do you mean? Create a custom validation such as number of digits, etc.?

Reply:
yes